blogposts/examples/how-to-write-blogposts.md

```diff
@@ -0,0 +1,638 @@
+
+[You can use numbers for reference-style link definitions][1]
+
+Or leave it empty and use the [link text itself].
+
+URLs and URLs in angle brackets will automatically get turned into links.
+www.example.com, https://example.com, and contact@example.com.
+
+Some text to show that the reference links can follow later.
+
+[arbitrary case-insensitive reference text]: https://www.mozilla.org
+[1]: http://slashdot.org
+[link text itself]: http://www.reddit.com
+```
+
+[I'm an inline-style link](https://www.google.com)
+
+[I'm an inline-style link with title](https://www.google.com "Google's Homepage")
+
+[I'm a reference-style link][Arbitrary case-insensitive reference text]
+
+[I'm a relative reference to another blogpost](/blog/example-1)
+
+[You can use numbers for reference-style link definitions][1]
+
+Or leave it empty and use the [link text itself].
+
+URLs and URLs in angle brackets will automatically get turned into links.
+www.example.com, https://example.com, and contact@example.com.
+
+Some text to show that the reference links can follow later.
+
+[arbitrary case-insensitive reference text]: https://www.mozilla.org
+[1]: http://slashdot.org
+[link text itself]: http://www.reddit.com
+
+## Images
+
+```
+Here's our logo (hover to see the title text), taken from the public folder of the repo:
+
+Inline-style:
+![alt text](/blogposts/preview/default.jpg "Logo Title Text 1")
+
+Reference-style:
+![alt text][logo]
+
+[logo]: /blogposts/preview/default.jpg "Logo Title Text 2"
+
+To change an images size use the `<img>` element:
+<img src="/blogposts/preview/default.jpg" width="200" height="100"></img>
+
+You can also use external links as images:
+![Minion](https://octodex.github.com/images/minion.png)
+![Stormtroopocat](https://octodex.github.com/images/stormtroopocat.jpg "The Stormtroopocat")
+
+Like links, Images also have a footnote style syntax
+
+![Alt text][id]
+
+With a reference later in the document defining the URL location:
+
+[id]: https://octodex.github.com/images/dojocat.jpg  "The Dojocat"
+```
+
+Here's our logo (hover to see the title text), taken from the public folder of the repo:
+
+Inline-style:
+![alt text](/blogposts/preview/default.jpg 'Logo Title Text 1')
+
+Reference-style:
+![alt text][logo]
+
+[logo]: /blogposts/preview/default.jpg 'Logo Title Text 2'
+
+To change an images size use the `<img>` element:
+<img src="/blogposts/preview/default.jpg" width="200" height="100"></img>
+
+You can also use external links as images:
+![Minion](https://octodex.github.com/images/minion.png)
+![Stormtroopocat](https://octodex.github.com/images/stormtroopocat.jpg 'The Stormtroopocat')
+
+Like links, Images also have a footnote style syntax
+
+![Alt text][id]
+
+With a reference later in the document defining the URL location:
+
+[id]: https://octodex.github.com/images/dojocat.jpg 'The Dojocat'
+
+## Footnotes
+
+```
+Footnote 1 link[^first].
+
+Footnote 2 link[^second].
+
+Inline footnote^[Text of inline footnote] definition.
```

Comment:
missing in actual render

Reply:
Could catch! It didn't work so I removed it, but forgot to remove it in the code block.